### ISSUE TITLE
Avoid using `sun-moon` icon to represent "system themes"

### ISSUE THREAD

**Issue by Ashvith10** [lucide-icons/lucide#1719]
### Package

- [ ] lucide
- [ ] lucide-angular
- [ ] lucide-flutter
- [ ] lucide-preact
- [ ] lucide-react
- [ ] lucide-react-native
- [ ] lucide-solid
- [ ] lucide-svelte
- [ ] lucide-vue
- [ ] lucide-vue-next
- [ ] Figma plugin
- [ ] all JS packages
- [X] site

### Description

If I am not wrong, I'm assuming that the symbol `sun-moon`, alongside `sun` and `moon` also serves it purpose to inform users that this particular web-app supports a three-state theme toggle - light, dark and system.

Personally, I think that this comes out as misleading, and also fails to inform users - at least to me on first glance, it seemed to indicate moonlight - therefore, "system theme" or "auto theme" is not what this should be representing, at least in my opinion.

For now, I am making use of the `monitor` icon to indicate this - and I think that this is a better replacement. I am not able to think of an alternative symbol that is also unique in nature, while also in guideline with the Lucide design principles.

[Screencast from 2023-12-10 17-17-25.webm](https://github.com/lucide-icons/lucide/assets/113123021/c98067ad-d46d-4f5a-ada8-19c6733d9661)

TL:DR;
- make monitor the new icon to indicate system themes (probably an easy fix - just change the tags)
OR
- create a new icon to depict system themes (will require some research)

### Use cases

This should improve the message conveyed by the icons.

### Checklist

- [X] I have searched the existing issues to make sure this bug has not already been reported.

**Comment by karsa-mistmere** [comment]
It's just an icon that people may use as they wish. While it has various tags, it doesn't have a prescribed use case, nor should it.

`contrast` is also often used for symbolising a "system" colour theme, I'm closing this issue as it has no clear objective, if and when someone comes up with a more appropriate system theme icon, they should submit it as an icon request.

**Comment by Ashvith10** [comment]
@karsa-mistmere fair point. But what is your opinion on adding suggestive tags to both `monitor` and `contrast` with respect to theme?

**Comment by karsa-mistmere** [comment]
We could certainly add some _generic_ tags to contrast, e.g. `color`, `theme`, `scheme`, not sure about monitor for that we could add `system`.

Fun find: the new angular.dev site actually uses this icon for the system theme.
![image](https://github.com/lucide-icons/lucide/assets/17746067/88c13f7b-027a-4a53-9036-7fe6210cf9bd)

**Comment by Ashvith10** [comment]
That is actually very similar to a [picture you've shared before](https://github.com/lucide-icons/lucide/pull/545#issuecomment-1095338659). Now, out of curiosity, I tried using the SunMoon component in SvelteKit, and I quite like this icon - which also happens to be very similar to the above design.

[Screencast from 2023-12-11 14-18-28.webm](https://github.com/lucide-icons/lucide/assets/113123021/3e6c717a-16f8-4b20-8115-2ea25cc61f95)

But now, here's the twist - the Lucide site has this other sun-moon logo that I was talking about - which I was talking with reference to, and the one that I also happen to not be in favor of.

![Screen Shot 2023-12-11 at 14 20 02](https://github.com/lucide-icons/lucide/assets/113123021/839c1659-6f7d-40d6-bb33-4f998284ae53)

Looks like there is some inconsistency between different frameworks? Which of the following is the deprecated one?
